src/app/(courseList)/page.tsx

```diff
@@ -0,0 +1,29 @@
+import { getCourseListing } from '@services/archive'
+import { getSessionUser } from '@services/tkoUserService'
+
+import CourseListItem from '@components/CourseList/CourseListItem'
+import CourseListHeader from '@components/CourseList/CourseListHeader'
+import NoCoursesFound from '@components/CourseList/NoCoursesFound'
+
+const Page = async () => {
+  const { rights } = await getSessionUser()
+  const courses = await getCourseListing()
+
+  return (
+    <div role="table" aria-label="Courses" className="list-container divide-y">
+      <CourseListHeader showManage={rights.remove || rights.rename} />
+      {courses.length === 0 && <NoCoursesFound />}
+      {courses.map(course => {
+        return (
```

Comment:
Since there's no logic here, I would personally prefer `=> <CourseListItem... />` instead, but this is a minor nit and can be justified either way.

Reply:
This could be done, I think that this is left from debugging. TODO: make prettier

Reply:
Made more prettier.